onyo/commands/edit.py

```diff
@@ -57,16 +98,19 @@ def edit(args, onyo_root):
         sys.exit(1)
 
     # check and set paths
-    list_of_sources = prepare_arguments(args.asset, onyo_root)
-    # iterate over file list, edit them, add changes
+    list_of_sources = prepare_arguments(args.asset, repo)
+    editor = get_editor(repo.root)
+
+    # change files and add changes to git
     for source in list_of_sources:
```

Comment:
Please retire the name "source". This is a copy/paste from `mv` and others. There is no "source" here. They are assets.

`list_of_sources` is needlessly verbose. Try:
```
for a in assets:
```